macos/FluentUI/Button/Button.swift

```diff
@@ -491,11 +541,19 @@ class ButtonCell: NSButtonCell {
 			break
 		}
 
+		// First, center the Title
 		var x = (rect.width - titleSize.width) / 2
 		var y = (rect.height - titleSize.height) / 2 + titleVerticalPositionAdjustment
 
 		if xOffsetSign != 0 {
+			// Second, offset the Title from the Primary Image
 			x += CGFloat(xOffsetSign) * (imageSize.width + titleToImageSpacing) / 2
+
+			// Third, offset the Title from the Secondary Image
```

Comment:
nit: you can omit the `First`, `Second` and `Third` parts of the comments as the control flow already dictates the order.